meson.build

```diff
@@ -186,7 +186,7 @@ install_data(scripts,
     install_dir: get_option('bindir'))
 
 package_datadir = join_paths('share', 'vpd')
-install_subdir('ibm_vpd/fru/', install_dir: package_datadir, strip_directory: true)
+install_subdir('config/ibm/', install_mode: 'rw-r--r--', install_dir: package_datadir, strip_directory: true)
```

Comment:
Install mode should be -rwxr-xr-x

Reply:
Oh Okay, need to check the log, I'll add execute. Thanks.